github/pull_request.rb

```diff
@@ -98,4 +100,8 @@ def replace_labels(remove_labels, add_labels)
   def add_comment(message)
     @client.add_comment(@repo, @number, message)
   end
+
+  private def redmine_url
```

Comment:
I recall you saying this syntax only works on Ruby 2.X+ ? If so, I think this would break the current implementation given the cartridge is running 1.9.3 at present.